packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -19,7 +19,10 @@ module.exports.modifyWebpackConfig = ({ config }, { compilerOptions }) => {
   const opts = { compilerOptions: copts, transpileOnly: true }
   config.loader(`typescript`, {
     test,
-    loader: `ts-loader?` + JSON.stringify(opts),
+    loaders: [
+      `babel?${JSON.stringify(config._loaders.js.config.query)}`,
```

Comment:
This may do more than you want here. You really _only_ want to run the babel query extractor plugin. This will run the file through all the babel transforms (admittedly most won't matter after tsc runs). 

Honestly you probably better to just leave this off entirely, the plugin isn't important to running anything, it's really more of a minification step, as the queries aren't used a runtime it's better to just remove the code.

another option is to move the babel plugin to a proper package that can be consumed by plugins directly